modal/cli/secret.py

```diff
@@ -48,32 +50,69 @@ async def list_(env: Optional[str] = ENV_OPTION, json: bool = False):
 @secret_cli.command("create", help="Create a new secret.")
 @synchronizer.create_blocking
 async def create(
-    secret_name,
-    keyvalues: list[str] = typer.Argument(..., help="Space-separated KEY=VALUE items"),
+    secret_name: str,
+    keyvalues: Optional[list[str]] = typer.Argument(default=None, help="Space-separated KEY=VALUE items."),
     env: Optional[str] = ENV_OPTION,
+    from_dotenv: Optional[Path] = typer.Option(default=None, help="Path to a .env file to load secrets from."),
```

Comment:
Should we try to mirror the `Secret.from_dotenv` method and allow this to be used just as a flag `(modal secret create whatevrer --from-dotenv` with auto-discovery of the file? It's not a huge problem if we don't do that, but I think it's how I'd expect it to work.

Reply:
Discussed on Slack, and this turned out to be harder than expected, so pushing it to a potential follow-up.